.github/workflows/create-release.yml

```diff
@@ -37,18 +37,27 @@ jobs:
           npm run exec -- ./scripts/release.ts --release-as ${{ github.event.inputs.mode }} --ci-mode
           echo "::set-output name=new_version::$(node -p "require('./package.json').version")"
 
+      - name: Generate Release Body
+        id: generare_body
+        run: |
+          npx extract-changelog-release > RELEASE_BODY.md
+          echo ::set-output name=tag_name::$(git describe HEAD --abbrev=0)
+          echo ::set-output name=release_body::$(cat RELEASE_BODY.md)
+
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v4
         with:
+          delete-branch: true
           token: ${{ secrets.TINKOFF_BOT_PAT }}
           branch: release/${{ steps.run-release.outputs.new_version }}
           title: 'Release/${{ steps.run-release.outputs.new_version }}'
+          commit-message: |
+            ${{ steps.generare_body.outputs.release_body }}
 
-      - name: Generate Release Body
-        id: generare_body
-        run: |
-          npx extract-changelog-release > RELEASE_BODY.md
-          echo ::set-output name=tag_name::$(git describe HEAD --abbrev=0)
+            > **DO NOT SQUASH OR REBASE ME**
```

Comment:
how about add this warning to `body` of pull-request?

Reply:
body of pull-request == commit-message

Reply:
fixed